CHANGELOG.md

```diff
@@ -2,3 +2,5 @@
 
 ## 1.0-21 (In Development)
 
+## Unreleased
+  - Now `NavigationService` with its rerouting, etc. events is handled not in `VisionARViewController`.
```

Comment:
Please consider https://keepachangelog.com/en/1.0.0/ and update change log.

Reply:
After discussion with @chezzdev, decided to not use the entire [Changelog format](https://keepachangelog.com/en/1.0.0/), but a few aspects from it.
Refactored.